### PR TITLE
Add news to home.

### PR DESCRIPTION
**Pull request by NistorCristian**
Signed-off-by: Nistor Cristian <contact@nistorcristian.com>

### CODE REVIEW

File: themes/navy/source/js/main.js [hunk 1]
```diff
@@ -1,5 +1,187 @@
...
+            type: 'ghost'
+        },
+        // {
+        //     url: 'https://news.keycard.tech',
+        //     key: 'd7ad0ac3484bdf01fadf32b59c',
+        //     version: 'v2',
+        //     limit: 3,
+        //     blog: 'keycard',
+        //     type: 'ghost'
+        // },
```
Comment (jakubgs) [inline themes/navy/source/js/main.js]: Why keep commented out stuff?

File: themes/navy/source/js/main.js [hunk 2]
```diff
@@ -1,5 +1,187 @@
...
+            type: 'ghost'
+        },
+        {
+            url: 'https://embark.status.im/atom.xml',
+            blog: 'embark',
+            type: 'xml'
+        }
+    ];
+
+    function loadGhostPosts(arr){
```
Comment (jakubgs) [inline themes/navy/source/js/main.js]: Any specific reason why you are defining this(or other) function within the `$(document).ready()`?

File: themes/navy/source/js/main.js [hunk 3]
```diff
@@ -1,5 +1,187 @@
...
+
+        return ghostAPI.posts
+            .browse({
+                include: 'authors',
+                fields: ['title', 'url', 'published_at'],
+                limit: arr.limit
+            })
+            .then(function(posts){
+
+                $.each(posts, function (i,v) {
```
Comment (jakubgs) [inline themes/navy/source/js/main.js]: Why are you abusing jQuerys `each()` so much? As far as I know the support of `for (const x of y)` is good:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
Only IE doesn't support it.

**Review by jakubgs — COMMENTED** [review]
I don't know if I like the appending of multiple JS libraries in `themes/navy/layout/layout.ejs`.

Using something like [Rollup.js](https://rollupjs.org/) could pack this all into one much smaller bundle that gets loaded with one request. You can see how that's done in our `nimbus-site` repo:
https://github.com/status-im/nimbus-site/blob/2e163e33/gulpfile.js#L37-L43
This repo explains how to use 2 plugins to roll external dependencies in:
https://github.com/rollup/rollup-starter-lib
I'm fine merging it without Rollup but it would be nice as a separate PR.

Also, I'd really appreciate some comments explaining what things like this do:
https://github.com/status-im/status-network-site/blob/bf5644982ca3e84e9de9da4f2b504d4f3ddb6a93/themes/navy/source/js/main.js#L12-L22

File: themes/navy/source/js/main.js (hunk 2 above)
Comment (NistorCristian) [reply themes/navy/source/js/main.js]: I put the functions outsite of `ready`

File: themes/navy/source/js/main.js (hunk 3 above)
Comment (NistorCristian) [reply themes/navy/source/js/main.js]: Replaced jQuery's `each` with `for`

File: themes/navy/source/js/main.js [hunk 4]
```diff
@@ -1,14 +1,113 @@
 $(document).ready(function () {
 
+    var allPosts = [],
```
Comment (jakubgs) [inline themes/navy/source/js/main.js]: If you define `allPosts` here it won't be available in `loadGhostPosts()`.

**Review by jakubgs — CHANGES_REQUESTED** [review]
If you want to get the authors for Ghost posts you need to add `&include=authors` to URL arguments:
* https://ghost.org/docs/api/v3/content/#posts
* https://ghost.org/docs/api/v3/content/#include

Otherwise the `authors` field is not included in the response. Abbreviated example:
```bash
curl -s 'https://our.status.im/ghost/api/v2/content/posts/?key=2abb4728e472593059746cc5e8&limit=1&include=authors'
```
```json
{
  "posts": [
    {
      "id": "5ddbb92e1858f500017675e1",
      "uuid": "29f6420b-9ab5-4906-8312-e433b316cfa7",
      "title": "Town Hall #48 - November 25, 2019",
      "slug": "town-hall-november-25-2019",
      "html": "<p>This weeks Town Hall we have updates from the Core Team on V1, Infra Updates, and the Pops team on Hiring. Also we have two special presentation from our UX Researcher Hester on payment networks in Venezuela &amp; the New Status Network - Enjoy!",
      "comment_id": "5ddbb92e1858f500017675e1",
      "feature_image": "https://our.status.im/content/images/2019/11/Screen-Shot-2019-11-25-at-17.07.49.png",
      "featured": false,
      "page": false,
      "created_at": "2019-11-25T11:21:18.000+00:00",
      "updated_at": "2019-11-25T16:37:17.000+00:00",
      "published_at": "2019-11-25T16:23:30.000+00:00",
      "twitter_image": "https://our.status.im/content/images/2019/11/Screen-Shot-2019-11-25-at-17.07.49-1.png",
      "twitter_title": null,
      "twitter_description": "This weeks Town Hall we have updates from, theCore Team on V1 launch, Infra, & the Pops team on Hiring - If you know anyone, let us know!\n\nWe also have two special presentation, one from our UX Researcher @hesterbruikman on Payment networks in Venezuela & the New Status Network- Tune in & Enjoy! ",
      "custom_template": null,
      "canonical_url": null,
      "authors": [
        {
          "id": "5bbe60cc4f622300016e6503",
          "name": "Rajanie ",
          "slug": "rajanie",
          "profile_image": "https://our.status.im/content/images/2018/08/IMG_3402.JPG",
          "cover_image": "https://our.status.im/content/images/2018/08/shutterstock-millennial-pink-background-6.jpg",
          "bio": null,
          "website": null,
          "location": "Berlin",
          "facebook": null,
          "twitter": null,
          "meta_title": null,
          "meta_description": null,
          "url": "https://our.status.im/author/rajanie/"
        },
      ],
      "primary_tag": null,
      "url": "https://our.status.im/town-hall-november-25-2019/",
      "excerpt": "This weeks Town Hall we have updates from the Core Team on V1, Infra Updates,\nand the Pops team on Hiring. Also we have two special presentation from our UX\nResearcher Hester on payment networks in Venezuela & the New Status Network -\nEnjoy! \n\nPeopleOps - Hiring \n\n\nCore Update - V1 Progress\nStatus-go v1 scope [https://github.com/orgs/status-im/projects/54]\nCore UI project board [https://github.com/orgs/status-im/projects/52]\n\n\n\nSecurity and Infrastructure Updates\nSlide link - here\n[https://docs."
    }
  ],
  "meta": {
    "pagination": {
      "page": 1,
      "limit": 1,
      "pages": 250,
      "total": 250,
      "next": 2,
      "prev": null
    }
  }
}
```

**Comment by jakubgs** [comment]
Oh wait, it is included in here:
https://github.com/status-im/status-network-site/blob/ce08d2f23c76b565a35e17afb28f7863f5c6ff0e/themes/navy/source/js/main.js#L168-L170
So the issue must be in the JS, because I'm seeing `undefined`:
the_status_network_undefined_authors

**Comment by jakubgs** [comment]
Found it:
https://github.com/status-im/status-network-site/blob/ce08d2f23c76b565a35e17afb28f7863f5c6ff0e/themes/navy/source/js/main.js#L178-L180
It's `v.name` instead of `a.name`.

**Review by jakubgs — APPROVED** [review]
I built it locally and it looks good and works fine.
```
 $ du -hsc public/js/*
344K	public/js/main.js
336K	public/js/main.min.js
680K	total
```
336KB isn't huge, but I guess we can improve it later with something smarter than just concatenation by using some tree shaking, but it's fine for now.

Please squash the commits into one and I'll merge.

**Comment by jakubgs** [comment]
I just remembered that the branch is here, so i took the liberty of squashing it myself.

**Comment by jakubgs** [comment]
Looks good to me: https://dev.statusnetwork.com/. Pushing to prod.

Thanks for all your work, appreciate it!

**Pull request merged**